docs - embedding a chart (#78856)
metabase/metabase: docs/embedding/sdk/dashboards.md

## Customizing drill-through question layout

> When drilling through or clicking on a question card in the dashboard, you will be taken to the question view. By default, the question is shown in the [default layout](../question.md#customize-the-layout-of-an-interactive-chart) for interactive questions.

Author: ```suggestion
Drilling through or clicking on a question card in the dashboard will take you to the question view with the [default layout](../question.md#customize-the-layout-of-an-interactive-chart) for interactive questions.
```

{% include_file "{{ dirname }}/snippets/dashboards/custom-drill-through-question-layout.tsx" snippet="example-2" %}
```

> The questionView prop accepts a React component that will be rendered in the question view, which you can build with namespaced components within the `InteractiveQuestion` component. See [customize the layout](../question.md#customize-the-layout-of-an-interactive-chart) for an example layout.

Author: ```suggestion
The `questionView` prop accepts a React component that will be rendered in the question view, which you can build with namespaced components within the `InteractiveQuestion` component. See [customize the layout](../question.md#customize-the-layout-of-an-interactive-chart).
```
